src/ontology/cl-edit.owl

```diff
@@ -31277,6 +31279,29 @@ AnnotationAssertion(rdfs:label obo:CL_4033037 "mucus secreting cell of tracheobr
 EquivalentClasses(obo:CL_4033037 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0007196) ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410077)))
 SubClassOf(obo:CL_4033037 obo:CL_0002202)
 
+# Class: obo:CL_4033038 (lung resident memory CD4-positive, alpha-beta T cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24064670") Annotation(oboInOwl:hasDbXref "PMID:31265968") obo:IAO_0000115 obo:CL_4033038 "An alpha-beta CD4 T cell that resides in the lung.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033038 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4033038 <https://orcid.org/0000-0002-9185-3994>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033038 "2023-04-27T10:15:45Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24064670") oboInOwl:hasExactSynonym obo:CL_4033038 "lung resident memory CD4 T cells")
```

Comment:
If using "cells" in the synonym, this would be a related synonym with type "plural".